templates/features.html

```diff
@@ -94,6 +94,12 @@
   search.value = location.hash.substr(1);
 }
 
+chromeMetadata.addEventListener('query-changed', function(e) {
+  search.value = '=' + e.detail.version;
+  var event = new Event('input');
```

Comment:
I was expecting that to dispatch. But I guess it doesn't when the value is programmatically changed. I will change it to filter it directly.